notebooks/learning_graph_structured_data_h2.ipynb

```diff
@@ -0,0 +1,835 @@
+{
```

Comment:
<p>Ditto:</p><ul><li>the disk appears as an ellipse, it would be cleaner if it could be a real disk.</li><li>Add some legend? Remind the reader that each node is a karate club member, the color is their appartenance to one of the two subgroups?</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/806/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
Done